opentracing/dapperish/dapper.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
this whole `dapperish` package is just an example implementation... obviously the code is not very useful, etc.